src/components/NotesMindMap/NotesMindMapComponent.js

```diff
@@ -97,33 +108,33 @@ export class NotesMindMapComponent extends Component {
         });
       } else {
         if (targetNote.id !== selectedNote.id) {
-          this.props.changeSelectedNote(targetNote);
+          this.props.changeSelectedNote({ note: targetNote, edges: this.props.edges });
```

Comment:
Does `changeSelectedNote` require `edges`?

Reply:
Passing to check if a note's children have been fetched.